### PR TITLE
Completed to_dict() method in image_layer class

### PR DESCRIPTION
**Pull request by Abhigupta4981**
Added diff_id and import_str properties in the dictionary returned by to_dict method

Signed-off-by: Abhi Gupta <abhigupta4981@gmail.com>

This PR addresses issue #86

### CODE REVIEW

**Comment by nishakm** [comment]
@Abhigupta4981 someone else asked to work on this first. If I don't hear back from them by 6:00 am PST, I'll merge your change.

File: classes/image_layer.py [hunk 1]
```diff
@@ -108,7 +108,9 @@ def to_dict(self):
             pkg_list.append(pkg.to_dict())
             layer_dict.update({self.fs_hash: {'packages': pkg_list,
                                               'tar_file': self.__tar_file,
-                                              'created_by': self.__created_by
+                                              'created_by': self.__created_by,
+                                              'diff_id': self.__diff_id,
+                                              'import_str': self.__import_str
                                               }})
```
Comment (nishakm) [inline classes/image_layer.py]: Would you mind using self.tar_file etc instead of the private name for the properties? Thanks!

Comment (Abhigupta4981) [reply classes/image_layer.py]: yeah ofcourse

File: classes/image_layer.py [hunk 2]
```diff
@@ -108,7 +108,9 @@ def to_dict(self):
             pkg_list.append(pkg.to_dict())
         layer_dict.update({self.fs_hash: {'packages': pkg_list,
                                           'tar_file': self.tar_file,
-                                          'created_by': self.created_by
+                                          'created_by': self.created_by,
+                                          'diff_id': self.diff_id,
+                                          'import_str': self.import_str
```
Comment (nishakm) [inline classes/image_layer.py]: This looks good. Thanks!

**Comment by nishakm** [comment]
@Abhigupta4981 could you move the `Signed-off-by: Abhi Gupta <abhigupta4981@gmail.com>` after the `This PR addresses issue #86` line in your commit message? Thank you!

**Comment by nishakm** [comment]
Looks good to me. Thanks for sticking with this!

**Pull request merged**